src/libraries/Microsoft.Extensions.Configuration.Xml/tests/ConfigurationProviderXmlTest.cs

```diff
@@ -20,6 +20,16 @@ public override void Combine_after_other_provider()
             // Disabled test due to XML handling of empty section.
         }
 
+        public override void Load_from_single_provider_with_duplicates_throws()
+        {
+            // Disabled test due to XML handling of duplicate keys section.
```

Comment:
Why not just remove those tests from the base? 

Is this a breaking change that we should put behind a flag?

Reply:
> Why not just remove those tests from the base?

These implementations disable/override the test:

- ConfigurationProviderCommandLineTest
- ConfigurationProviderEnvironmentVariablesTest
- ConfigurationProviderIniTest
- ConfigurationProviderJsonTest
- **ConfigurationProviderXmlTest** (since this PR)

These implementations don't override it:

ConfigurationProviderMemoryTest
ConfigurationProviderIniTest

So it looks like there's still two providers that throw an exception for duplicate keys within a single provider.

> Is this a breaking change that we should put behind a flag?

The only way in which this is breaking, is that it no longer throws an exception where it previously did. If that is considered a breaking change, then maybe a flag is appropriate. IMHO I don't think it should be a flag.

Reply:
> So it looks like there's still two providers that throw an exception for duplicate keys within a single provider.

Ah ok, I missed the fact that there was still two providers that threw an exception. 

> If that is considered a breaking change.

I confirmed and we don't consider that a breaking change.